lib/splitclient-rb/engine/matchers/semver.rb

```diff
@@ -6,22 +6,33 @@ class Semver
     PRE_RELEASE_DELIMITER = '-'
     VALUE_DELIMITER = '.'
 
-    attr_reader :major, :minor, :patch, :pre_release, :is_stable, :old_version
+    attr_reader :major, :minor, :patch, :pre_release, :is_stable, :old_version, :version
 
-    #
-    # Class Initializer
-    #
-    # @param version [String] raw version as read from splitChanges response.
     def initialize(version)
       @major = 0
       @minor = 0
       @patch = 0
       @pre_release = []
       @is_stable = false
       @old_version = version
```

Comment:
remove `old_version`, it is not necessary now.

Reply:
will do